spec/forms/work_form_spec.rb

```diff
@@ -53,12 +54,37 @@
     end
   end
 
+  describe 'contributors validation' do
+    let(:contributor_error) { { contributors: ['Please add at least one contributor.'] } }
+
+    context 'with no contributors' do
+      it 'does not validate' do
+        form.valid?
+        expect(form.errors.messages).to include(contributor_error)
+      end
+    end
+
+    context 'with contributors' do
+      let(:form_with_contributors) { described_class.new(work_with_contributors) }
```

Comment:
Could you replace this line with:

```ruby
  let(:work) { build(:work, :with_contributors) }
```

and then you can use the form object above which should be lazily evaluated with the "right" work here, rather than creating a parallel structure? if so, then you can also dump the `work_with_contributors` `let` statement above.

Reply:
sounds good, done